### ISSUE TITLE
Blocks left click on some websites

### ISSUE THREAD

**Issue by the-moog** [lunu-bounir/allow-right-click#31]
Great utility and is most useful. But I find that it also block left click on some websites.
Weirdly not all controls though. I've yet to come up with a concise way to repeat this though.

As once you enable it you can't then disable it, it's difficult to tell if it's the site or the plugin.. Though I suspect the latter as the site works again if you then turn it off for that tab.

**Comment by nerdybuff** [comment]
Yeah true with most websites left click gets blocked when it's activated. I liked to have it auto-engage with some websites but I had to stop because it interfered with some of the site's functionality.
If you'd like more info please contact me!

**Comment by lunu-bounir** [comment]
You need to either provide a sample web page with the issue or find a solution and ask for pull merge.
